refresh/templates/common/Metrics.swift

```diff
@@ -2,21 +2,25 @@ import Kitura
 import SwiftMetrics
 import SwiftMetricsDash
 import SwiftMetricsPrometheus
+import SwiftMetricsREST
 import LoggerAPI
 
 var swiftMetrics: SwiftMetrics?
 var swiftMetricsDash: SwiftMetricsDash?
 var swiftMetricsPrometheus: SwiftMetricsPrometheus?
+var swiftMetricsRest: SwiftMetricsREST?
 
 func initializeMetrics(router: Router) {
     do {
         let metrics = try SwiftMetrics()
         let dashboard = try SwiftMetricsDash(swiftMetricsInstance: metrics, endpoint: router)
         let prometheus = try SwiftMetricsPrometheus(swiftMetricsInstance: metrics, endpoint: router)
+        let rest = try SwiftMetricsREST(swiftMetricsInstance : metrics, endpoint: router)
```

Comment:
nit: space after `swiftMetricsInstance`

Reply:
Fixed, commit amended to keep history tidy.